game/scripts/vscripts/components/points/game_length_vote.lua

```diff
@@ -47,4 +47,7 @@ function GameLengthVotes:SetGameLength ()
   end
   DebugPrint ( 'votes ' .. votes.short .. ', ' .. votes.normal .. ', ' .. votes.long .. ' result: ' .. length)
   GameRules.GameLength = length
+  CustomGameEventManager:Send_ServerToAllClients("gamelength_vote_confirmed", {
```

Comment:
Not too sure what hierarchy you mean. I think it makes more sense to set table values relating to length in the `GameLengthVotes` module anyway, though the score initialisation should probably stay in int `PointsManager`.

Numbers can be very easily converted to text using `switch` or some other solution. I'm guessing you meant that it would require changing if the length and kill limit relation was ever changed.